seneca/smart_contract_user_libs/storage/kv.py

```diff
@@ -6,8 +6,151 @@
 # * type info
 # always upsert
 
-exports = None
+import seneca.seneca_internal.storage.easy_db as db
+from seneca.seneca_internal.storage.mysql_base import get_str_to_py_cast_func
+import datetime
+
+ex = None
+name_space = 'test_tabular'
```

Comment:
I don' think we want to do this. The name_space should be explicitly set by the caller.

Reply:
corrected in latest version of this branch